.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -5,23 +5,23 @@
 <!-- Make sure that your PR is not a duplicate -->
 
 #### Pre-Submission Checklist
-<!-- Go over all points below, and put an `x` in all the boxes that apply. -->
-<!-- All points should be checked, otherwise, read the CONTRIBUTING guidelines from above-->
+<!-- Go over all points below, and after creating the PR click on all the check boxes that apply. -->
```

Comment:
~~, and after creating the PR click on all~~ , and after creating the PR, click on all